css/index.css

```
@@ -0,0 +1,102 @@
+body {
+  font-family: sans-serif;
+  background-color: #FFEEDE;
+  ;
+}
+
+.slide__foodSpin {
```

No veo una clase `.slide` para que hagas un elemento que se llame `__foodSpin`. Tambien, se ve mas elegante separar las `palabras` con `-`
```suggestion
.slide-food-spin {
```